src/Patterns/dictionary.json

```diff
@@ -578,5 +578,31 @@
 				}
 			]
 		}
+	},
+	{
+		"name": "Featured Category Cover Image",
+		"slug": "woocommerce-blocks/featured-category-cover-image",
+		"images_total": 1,
+		"images_format": "landscape",
+		"content": {
+			"titles": [
+				{
+					"default": "100% natural denim",
+					"ai_prompt": "A description for a product"
```

Comment:
Just a question on the prompts in general, do we ever specify a text limit on the values returned to ensure they don't break designs?

Reply:
Great question: we are still fine-tuning the prompts to AI as part of a separate task :)